app/main.py

```diff
@@ -1,6 +1,25 @@
+import os
+import requests
+
+
+API_KEY = os.getenv("WEATHER_API_KEY")
+BASE_URL = "http://api.weatherapi.com/v1"
+FILTERING = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    payload = {
+        "key": API_KEY,
+        "q": FILTERING,
+    }
+    res = requests.get(BASE_URL + "/current.json", params=payload)
```

Comment:
It's a good practice to check the response status code before attempting to parse the JSON. Consider adding error handling to manage unsuccessful requests (e.g., `res.raise_for_status()` or checking `res.status_code`).